src/test/resources/simple-fail/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: v1
+name: app-fail
+version: 0.1.0
```

Comment:
a mock chart that does not exist in the mocked repo. this is used to test a failed verification